health-services/project/src/main/java/org/egov/project/config/ProjectConfiguration.java

```diff
@@ -186,4 +189,9 @@ public class ProjectConfiguration {
     @Value("${project.staff.attendance.topic}")
     private String projectStaffAttendanceTopic;
 
+    @Value("${project.task.resource.quantity.pattern}")
+    private String projectTaskResourceQuantityPattern;
```

Comment:
is this needed now?

Reply:
Removed invalid-error-message and task.resource.quantity.pattern properties